content/installation/from-packages.md

```diff
@@ -1,70 +1,84 @@
 ---
 layout: bt_wiki
-title: Installing using packages
+title: Installing from packages
 category: Installation
 draft: false
 weight: 200
 
 ---
 
-Compiled binaries are provided for Windows and RHEL based distributions. You can use these to easily install Cloudify.
+Cloudify packages greatly reduce the initial complexity of the installation process
+and are the recommended way of installing for new users.
+
+This installation method doesn't require internet connection during the install
+process.
+
+{{% gsNote title="Installation Environment" %}}
+Note that a virtualenv will be automatically created during installation and Cloudify's CLI will be installed within it.
```

Comment:
you can remove "Cloudify's CLI" and just use "Cloudify" i.e. we "install cloudify".